doc/sphinx/.sphinx/spellingcheck.yaml

```diff
@@ -27,3 +27,6 @@ matrix:
       - div.visually-hidden
       - img
       - a.p-navigation__link
+      - li:matches(.toctree-l3, .toctree-l2) # ignore table of content tags
+      - div:matches(.mermaid)                # ignore mermaid.js diagrams
```

Comment:
Not sure we should be ignoring those in the long run. They're actually meant to be somewhat human-readable even in text form.

But it's probably fine to start with.

There's some terms that got added to the wordlist, then, that can be removed.

Reply:
The problem is that these tags are included in every html file, so they fill the output with duplicate lines.